governance/votes/Executive vote - April 18, 2024.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Template - [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Reward, Aave Revenue Share Transfer, Whitelist New Address in the RWA015-A Output Conduit, Push USDP out of Input Conduit, Trigger Spark Proxy Spell - April 18, 2024
+summary:  March 2024 AD Compensation, Q1 2024 AVC Member Participation Reward, Spark Protocol-Aave Share Transfer, Whitelisting new address in the RWA015-a output conduit, pushing 84,211.27 USDP out of the MCD PSM PAX input conduit jar, triggering a Spark proxy spell on Ethereum and Gnosis.
```

Comment:
:green_circle: **nit. non-blocking**

This isn't really a summary, rather it is a repeat of the title. It could also benefit from using active voice. Suggested improvement below.

```suggestion
summary:  Distribute AD compensation for March 2024 and AVC Member participation rewards for Q1 2024, transfer Dai to Aave as part of the SparkLend revenue sharing agreement, whitelist a new address in the RWA015-a output conduit, convert USDP to Dai and deposit to the Surplus Buffer, trigger a Spark proxy spell on Ethereum and Gnosis.
```